starlight_help/src/content/docs/import-from-mattermost.mdx

```diff
@@ -43,31 +43,27 @@ Replace `<username>` and `<server_ip>` with the appropriate values below.
 <Tabs>
   <TabItem label="Default installation">
     <Steps>
-      1. SSH into your Mattermost production server.
-         ```bash "<username>" "<server_ip>"
-         ssh <username>@<server_ip>
+      1. Make sure you have [mmctl](https://github.com/mattermost/mmctl) installed - these
+         instructions assume your version is `7.5.1` or higher.
+      1. Log into your Mattermost production server.
+         ```bash "yourdomain"
+         mmctl auth login `<server_ip>`
          ```
-      1. Navigate to the directory which contains the Mattermost executable.
-         On a default install of Mattermost, the directory is `/opt/mattermost/bin`.
+         You’ll be prompted for a username (use your admin user), password, and for a
+         connection name. The connection name can be anything you want.
+      1. Create a full export of the server, including attached files.
          ```bash
-         cd /opt/mattermost/bin
+         mmctl export create
```

Comment:
https://docs.mattermost.com/administration-guide/manage/cloud-data-export.html#migrate-from-self-hosted-to-cloud says to use `--attachments` here. I assume we should add that here?